app/controllers/shopping_list_item_statuses_controller.rb

```diff
@@ -21,6 +21,7 @@ def activate_from_search
   def deactivate
     # crosses an item off of the list
     @shopping_list_item.deactivate!
+    @remove_aisle = @shopping_list_item.list.items.active.where(aisle: @shopping_list_item.aisle).blank?
```

Comment:
@mikevallano I don't love this approach. What I don't love:
* creating an instance variable with a boolean value
* containing this logic in a controller action
* subsequent logic of javascript if/else to make DOM element removal decisions


I would have replaced a whole aisle section partial except that the aisle section data is... tricky. See: line 30 app/views/shopping_lists/show.html.erb.